meson.build

```diff
@@ -0,0 +1,40 @@
+project(
+    'agrirouter-api-cpp'
+    , 'c'
+    , 'cpp' 
+    , meson_version : '>=0.46.0'
+    , default_options : [
+        'buildtype=minsize'
+        , 'warning_level=1'
+        , 'cpp_std=c++11'
+    ]
+)
+
+subdir('lib/AgrirouterClient')
+
+agrirouter_client_tester_include_dir = include_directories('./AgrirouterClientTester/inc')
+
+agrirouter_client_tester_sources = [
+    './AgrirouterClientTester/src/Application.cpp',
+    './AgrirouterClientTester/src/Communicator.cpp',
+    './AgrirouterClientTester/src/main.cpp'
+]
+
+if get_option('prod')
+    message('production build')
+    linker_flags=['-s', '-lcurl', '-luuid']
+else 
+    message('debug build')
+    linker_flags=['-lcurl', '-luuid']
+endif
```

Comment:
- you can use `dependency('libcurl')` and `dependency('uuid')` here instead of manually coding the linker flags, which may not work if libcurl/uuid are installed to non-default locations.
- `-s` can be handled by setting the global option `-Doptimization=s`, see https://mesonbuild.com/Builtin-options.html#core-options

Reply:
I have added the dependencies. 
I leave the strip flag in the project. 
The default striped flag can be used and I leave the addition also in the project, so I can use the same parameter in the agrirouterclient project and in the lib.